z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-control-plane/tasks/main.yaml

```diff
@@ -123,3 +122,36 @@
     PYTHONWARNINGS: 'ignore::UserWarning'
   when:
   - disk_type == "scsi"
+  - use_storage_provider == false
+  
+- name: Create BFV control volume from backend storage provider
+  os_volume:
+    state: present
+    name: "{{ item.1 }}-{{ item.0 }}-bfv"
+    image: "rhcos"
+    size: "{{ master_flavor_size.stdout_lines[0]}}"
+    volume_type: "{{ volume_type_template }}"
+    metadata: "{{ cluster_id_tag }}"
+    timeout: 3600
+  with_indexed_items: "{{ [os_cp_server_name] * os_control_nodes_number }}"
+  when: 
+  - use_storage_provider == true
+
+- name: 'Create the control server with BFV volume'
+  os_server:
+    name: "{{ item.1 }}-{{ item.0 }}"
+    flavor: "{{ os_flavor_master }}"
+    boot_volume: "{{ item.1 }}-{{ item.0 }}-bfv"
+    auto_ip: no
+    availability_zone: "{{ create_server_zone }}"
+    timeout: 2400
+    userdata: "{{ lookup('file', [item.1, item.0, 'ignition.json'] | join('-')) | string }}"
+    nics:
+    - port-name: "{{ os_port_master }}-{{ item.0 }}"
+    meta: "{{ cluster_id_tag }}"
+    terminate_volume: True
+  with_indexed_items: "{{ [os_cp_server_name] * os_control_nodes_number }}"
+  environment: 
```

Comment:
what's the reason of this?

Reply:
it's a circular to create OCP master nodes, the node count is defined in inventory.yaml.